src/impl/collection_notifier.hpp

```diff
@@ -49,6 +49,16 @@ struct TransactionChangeInfo {
     std::vector<ListChangeInfo> lists;
     std::vector<CollectionChangeBuilder> tables;
     bool track_all = false;
+
+    // Required by GCC 4.9
```

Comment:
Can we guard this so only gcc 4.9 sees it? See for example https://gcc.gnu.org/onlinedocs/cpp/Common-Predefined-Macros.html